lib/beaker-hostgenerator/data.rb

```diff
@@ -623,6 +623,15 @@ def osinfo
             'template' => 'redhat-7-x86_64'
           }
         },
+        'redhatfips7-64' => {
+          :general => {
+            'platform'           => 'redhat-fips-7-x86_64',
```

Comment:
@ScottGarman have you tried a beaker run using this platform? I'd be surprised if it didn't fail, as beaker's Platform object assumes that the platform string can be nicely split on `-` characters in this pattern
```
<os_name>-<os_version>-<arch>
```

If you'd like to have a 2-part `os_name` section of the platform string, I would suggest using an `_` like [cisco nexus in this example](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/data.rb#L201).

Reply:
Very useful feedback @kevpl, thanks!